client/src/components/forms/DepositBoxForm/DepositBoxForm.tsx

```diff
@@ -0,0 +1,250 @@
+import { Checkbox, InputNumber, Space, Typography } from 'antd';
+import s from './DepositBoxForm.module.less';
+const { Title, Text } = Typography;
+import { DepositColumn } from '../DepositFormColumn';
+import { InputNumberBox } from '../InputNumberBox';
+import { Content } from 'antd/lib/layout/layout';
+import { FormButton } from '../FormButton';
+import TextArea from 'antd/lib/input/TextArea';
+import { useState } from 'react';
+
+const moneyValues = {
```

Comment:
You can add as const after object inicialization, so no one can change it's values